src/widgets/WeatherWidget.js

```diff
@@ -16,7 +16,7 @@ class WeatherWidget extends Component {
     const device = this.props.device;
     switch (device.dtype) {
       case "Temp" :
-        return { temp: device.svalue1 };
+        return { temp: Number(device.svalue1).toFixed(1) };
       case "Temp + Humidity" :
         return { temp: device.svalue1, hum: device.svalue2 };
       case "Temp + Humidity + Baro" :
```

Comment:
Please declare the rounded temperature in a const before the switch:
const temp = Number(device.svalue1).toFixed(1);

Then replace device.svalue1 by temp in all three cases of the switch. 
This way the temperature will be rounded for all 3 device types.